ocs_ci/ocs/constants.py

```diff
@@ -1233,3 +1237,21 @@
 
 OCS_TAINT = "node.ocs.openshift.io/storage"
 VOLUMESNAPSHOT = "volumesnapshot"
+
+ROOK_CEPH_CONFIG_VALUES = f"""
+    [global]
```

Comment:
This will create 4 space indentation in the resulting config file. If that creates a problem, we can use `textwrap.dedent()` function to compenstate for that, see eg. https://github.com/red-hat-storage/ocs-ci/blob/c4d5cd540436e09ee31dd809b24d2d0d57ef6f7d/ocs_ci/ocs/fio_artefacts.py#L72

Reply:
thanks Martin. It is not needed, the configmap is set correctly and Ceph is deployed with debug log level. 
See https://github.com/red-hat-storage/ocs-ci/pull/3908#issuecomment-790930779